service/service.go

```diff
@@ -36,6 +37,12 @@ import (
 	"go.opentelemetry.io/collector/service/telemetry"
 )
 
+var ConnectorsFeatureGate = featuregate.GlobalRegistry().MustRegister(
```

Comment:
Can this be moved to some `internal` package to avoid having it be part of the public API?

Reply:
That's a good idea. I'll move it.